README.md

```diff
@@ -22,6 +23,8 @@ Or install it yourself as:
 
 ## Usage
 
+In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/numo/sparse`. To experiment with that code, run `bin/console` for an interactive prompt.
+
```

Comment:
In this section, we will write how to use the library instead of how to use this repository.
So this paragraph shouldn't move from the original position.
Or, we can completely remove this paragraph, I think.